src/app/workflow-popup/workflow-popup.component.css

```diff
@@ -0,0 +1,137 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+
+.top-header-hr {
+  height: 4px;
+  margin: 0;
+  background-color: #047cc0;
+  width: 100%;
+  border: none;
+  flex: none;
+}
+
+.top-hr:hover {
+  background-color: #009bef;
+}
+
+.bottom-header-dark {
+  background-color: #1a354d;
+}
+
+.bottom-header-hr {
+  margin: 0 2rem;
+  height: 1px;
+  background-color: #bdbabd;
+  border: none;
+  flex: none;
+}
+
+.bottom-header-hr-dark {
+  background-color: #516577;
+}
+
+.popup-window-header-area {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex: none;
+  height: 50px;
+  color: #777677;
+}
+
+.popup-window-header-area-dark {
+  background-color: #1a354d;
+  color: #C0BFC0;
+}
+
+.popup-window-header {
+  margin-left: 2rem;
+  font-size: 21px;
+  float: left;
+  flex: 1 1 auto;
+  display: flex;
+  align-items: center;
+}
+
+.close-button {
+  height: 20px;
+  width: 20px;
+  margin: 10px;
+  padding: 0;
+  flex: none;
+  box-sizing: border-box;
+  background-color: transparent;
+  transition: 0.5s;
+  background: url(../../../../zlux-widgets/src/app/assets/images/close_normal.png) right top no-repeat;
```

Comment:
You must not import images from another repo using relative paths.

NOTE: zlux-widgets is not even in the zlux superproject (nor should it be, it is intended to be imported as an npm package), so this import should fail for most developers.

Reply:
In terms of your second point, with the image assets pointing to the zlux-widgets repository, that was a mistake. These should be pointing to the shared folder that houses these assets through the inclusion of the zlux-widgets dependency. If they are not there for whatever reason, then it would be wise to add them into the assets folder of Workflow along with all the other UI assets.